packages/flutter/lib/src/services/system_channels.dart

```diff
@@ -12,28 +12,159 @@ class SystemChannels {
   SystemChannels._();
 
   /// A JSON [MethodChannel] for navigation.
+  ///
+  /// The following incoming methods are defined for this channel (registered
+  /// using [MethodChannel.setMethodCallHandler]):
+  ///
+  ///  * `popRoute`, which is called when the system wants the current route to
+  ///    be removed (e.g. if the user hits a system-level back button).
+  ///
+  ///  * `pushRoute`, which is called with a single string argument when the
+  ///    operating system instructs the application to open a particular page.
+  ///
+  /// See also:
+  ///
+  ///  * [WidgetsBindingObserver.didPopRoute] and
+  ///    [WidgetsBindingObserver.didPushRoute], which expose this channel's
+  ///    methods.
   static const MethodChannel navigation = const MethodChannel(
       'flutter/navigation',
       const JSONMethodCodec(),
   );
 
   /// A JSON [MethodChannel] for invoking miscellaneous platform methods.
   ///
-  /// Ignores missing plugins.
+  /// The following outgoing methods are defined for this channel (invoked using
+  /// [OptionalMethodChannel.invokeMethod]):
+  ///
+  ///  * `Clipboard.setData`: Places the data from the `text` entry of the
+  ///    argument, which must be a [Map], onto the system clipboard. See
+  ///    [Clipboard.setData].
+  ///
+  ///  * `Clipboard.getData`: Returns the data from the argument, a [String],
```

Comment:
I don't understand what this sentence is supposed to tell me.

Reply:
Should it maybe be "from the system clipboard"?

Reply:
fixed.